tests/test_decoder.py

```diff
@@ -714,17 +714,17 @@ def test_end_comment():
 
 
 def test_parse_error():
-    with pytest.raises(pvl.decoder.ParseError):
```

Comment:
Ok I agree.  I'll implement the ``mode`` in a couple commits. But I think instead of ``mode='strict'`` it should be a boolean like ``strict=True``. I think that will be easier to remember and still as clear as ``mode``